modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+== Deploying the Lokistack
```

Comment:
I only see one instance of "Lokistack" in this PR. Do you mean "Loki stack" or "LokiOperator"?

Reply:
Loki Operator is the Operator name. 
LokiStack is the schema for the lokistacks API. 
Loki is the component name / upstream project's name.